tests/behaviour/typeql/language/steps.rs

```diff
@@ -400,4 +402,9 @@ generic_step_impl! {
             matched entries of given {actual_answers}."
         );
     }
+
+    #[step(expr = "set time-zone is: {word}")]
+    async fn time_zone_info(context: &mut Context, timezone: String) {
+        env::set_var("TZ", timezone);
```

Comment:
Is this really the best way to change locale? I'd have expected a `setlocale()` call instead.

Reply:
Interesting. I tried searching for this function, but could not find a way to change timezone. The way I performed this in Rust is analogous to the way I did it in Client Python and Client Java: changing the system timezone by changing the environment variable TZ.